plasmapy/simulation/particle_tracker.py

```diff
@@ -285,9 +285,31 @@ def save_now(self):
 
 
 class ParticleTracker:
-    """
-    General particle tracer.
+    r"""A many body particle tracker for electric and magnetic fields with no particle-particle interactions.
+
+    Particles are created and pushed through calculated E and B fields. These fields are specified as part of a grid
+    which can then be interpolated to determine local field strength. The time step used can be specified, otherwise an
+    adaptive time step is calculated based off grid resolution and the particle's gyroradius.
+
+    The simulation is terminated when a specified stop condition is met. The results of a simulation can be exported
+    using a save routine.
 
+    Parameters
+    ----------
+    grids : `~plasmapy.plasma.grids.AbstractGrid` or subclass thereof, or list
+        of same.
+        A Grid object or list of grid objects containing the required
```

Comment:
An instance of `link to grid class`

For req_quantities I think you can just discuss all of this under the `req_quantities` keyword below